### DISCUSSION TITLE
Комбинированное поле для отображения названия

### DISCUSSION THREAD

**Discussion by Ard2p** [moonshine-software/moonshine#3] [Ideas]
У нас есть BelongsTo(select) и 3 поля в базе данных lastname, firstname, middlename.
Мне нужно вывести в lable все 3 поля но указать я могу только 1 name или через ресурс $titleField = 'name'; 

Нужно сделать комбинированное выводимое поле, я вижу 3 варианта:
1. Сделать чтобы в поле нейм можно было указывать правило примерно как в валидации, чтобы можно было строить конструкции для вывода
2. Сделать переопределяемую функцию для построения name
3. Сделать промежуточное поле где мы напишем php код или паттерн по которому мы скомбинируем поля и укажем в связи

**Comment by lee-to** [comment]
Добавил эту возможность в релиз v1.3.0 (Третий параметр у полей теперь также принимает анонимную функцию для генерации более сложных тайтлов)

`
BelongsTo::make('Страна', 'country', fn($item) => "$item->id.) $item->name")
`
